charts/newrelic-prometheus-agent/values.yaml

```diff
@@ -440,3 +440,6 @@ config:
   # This configuration should be used as a workaround whenever kubernetes and static job do not cover a particular use-case.
   # @default -- `[]`
   extra_scrape_configs: []
+
+  metric_type_override:
+    enabled: true
```

Comment:
shouldn't this be enabled only if integrations_fiters are enabled?

Reply:
Does it make sense to be able to deactivate this relabels if one of the integrations that need it is added?
I understand that is an overhead because that list can be overwritten and overcomplicating the helper could be a mess.

Reply:
we synched offline, we changed the scope making it similar to lowdatamode.

I think that this "fix_metric_option" should be always applied. I added the boolean just in case someone don't want that and to simplify unitTest